using_images/s2i_images/java.adoc

```diff
@@ -115,89 +128,478 @@ or from binary artifacts.
 ----
 endif::openshift-online[]
 
 ifndef::openshift-online[]
+[source,terminal]
 ----
-$ oc new-app registry.redhat.io/redhat-openjdk-18/openjdk18-openshift~<git_repo_URL> --context-dir=<context-dir> --build-env='ARTIFACT_DIR=relative/path/to/artifacts/dir' --build-env='MAVEN_ARGS=install -pl <groupId>:<artifactId> -am'
+$ oc new-app registry.redhat.io/redhat-openjdk-18/openjdk18-openshift~<git_repo_URL> --context-dir=<context_dir> --build-env='MAVEN_S2I_ARTIFACT_DIRS=relative/path/to/artifacts/dir' --build-env='MAVEN_ARGS=install -pl <groupId>:<artifactId> -am'
 ----
 endif::openshift-online[]
 
 [[s2i-images-java-deploy-applications-from-binary]]
 == Building and Deploying from Binary Artifacts
 
-The Java S2I builder image can be used to build an application using binary
-artifacts that you provide.  To do so, first create a new binary build using the
-`oc new-build` command:
+You can use the Java S2I builder image to build an application using binary
+artifacts that you provide.
 
+. Create a new binary build:
++
 ifdef::openshift-online[]
+[source,terminal]
 ----
-$ oc new-build --name=<application-name> redhat-openjdk18-openshift --binary=true
+$ oc new-build --name=<application_name> redhat-openjdk18-openshift --binary=true
 ----
 endif::openshift-online[]
 
++
 ifndef::openshift-online[]
+[source,terminal]
 ----
-$ oc new-build --name=<application-name> registry.redhat.io/redhat-openjdk-18/openjdk18-openshift --binary=true
+$ oc new-build --name=<application_name> registry.redhat.io/redhat-openjdk-18/openjdk18-openshift --binary=true
 ----
 endif::openshift-online[]
 
-Next, start a build using the `oc start-build` command, specifying the path to
-the binary artifacts on your local machine:
-
+. Start a build and specify the path to the binary artifacts on your local machine:
++
+[source,terminal]
 ----
-$ oc start-build <application-name> --from-dir=/path/to/artifacts --follow
+$ oc start-build <application_name> --from-dir=/path/to/artifacts --follow
 ----
 
-Finally, use the `oc new-app` command to create an application:
-
+. Create an application:
++
+[source,terminal]
 ----
-$ oc new-app <application-name>
+$ oc new-app <application_name>
 ----
 
+[[s2i-images-java-environment-variables]]
+== Java Environment Variables
+
+The following tables provide a comprehensive list of Java environment variables used for configuring the behavior of the OpenJDK containers.
+
+.Configuration environment variables
+[options="header"]
+
+|===
+|Variable Name |Description |Example Value
+
+|`*AB_JOLOKIA_CONFIG*`
+|If set, uses this file (including path) as Jolokia JVM agent properties (as described in Jolokia's link:https://www.jolokia.org/reference/html/agents.html#agents-jvm[reference manual]). If not set, the `/opt/jolokia/etc/jolokia.properties` will be created using the settings as defined in the manual. Otherwise, the rest of the settings in this document are ignored.
```

Comment:
I would suggest reworking this descriptions without the parentheses. Applies to whole table.

s/in Jolokia’s/in the Jolokia

s/will be created/is created